app/src/ui/diff/text-diff.tsx

```diff
@@ -240,11 +240,11 @@ const defaultEditorOptions: IEditorConfigurationExtra = {
     'Shift-Tab': false,
     // Steal the default key binding so that we can launch our
     // custom search UI.
-    [__DARWIN__ ? 'Cmd-F' : 'Ctrl-F']: showSearch,
+    'CmdOrCtrl+F': showSearch,
     // Disable all other search-related shortcuts so that they
-    // don't interfer with global app shortcuts.
-    [__DARWIN__ ? 'Cmd-G' : 'Ctrl-G']: false, // findNext
-    [__DARWIN__ ? 'Shift-Cmd-G' : 'Shift-Ctrl-G']: false, // findPrev
+    // don't interfere with global app shortcuts.
+    'CmdOrCtrl+G': false, // findNext
+    'Shift+CmdOrCtrl+G': false, // findPrev
```

Comment:
I don't believe any of these changes produce the desired results. To my knowledge CodeMirror doesn't support `CmdOrCtrl`, that's an electron/desktop-ism. We rely on CodeMirror's [default keymap](https://github.com/codemirror/CodeMirror/blob/c74a1cafc01a7e34af1b19dd4c82ff821c2e1442/src/input/keymap.js#L19-L47) so the entries in `extraKeys` need to match exactly what's in the `pc` or `mac` keymap, hence the inline conditionals.

I think we need to revert these changes save for the typo.